### DISCUSSION TITLE
Welcome to Conductor Discussions!

### DISCUSSION THREAD

**Discussion by apanicker-nflx** [Netflix/conductor#2165] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with the community.

**Comment by maheshyaddanapudi** [comment]
Now we have Spring Boot compatible Conductor Server - Yay ! to Version 3.X

**Comment by sunnycalsoft** [comment]
How can i know whether conductor server is up and running?

**Reply by kishorebanala** [reply]
https://github.com/Netflix/conductor#development
